src/app/components/SwapWidget/SwapStepTwo.jsx

```diff
@@ -20,21 +20,25 @@ import { retrievePairData } from 'Actions/rate'
 import { retrieveSwap, refreshSwap } from 'Actions/swap'
 import { getSwap } from 'Selectors/swap'
 import { getRateMinimumDeposit, getRatePrice } from 'Selectors/rate'
+import { getBip21Prefix } from 'Selectors/asset'
 import DataLayout from 'Components/DataLayout'
 
 import { container, qr, scan, receipt } from './style.scss'
 
 /* eslint-disable react/jsx-key */
 const SwapStepTwo = ({
   swap, handleRef, handleFocus, handleCopy, handleTimerEnd, secondsUntilPriceExpiry, 
-  minimumDeposit, estimatedRate,
+  minimumDeposit, estimatedRate, bip21Prefix
 }) => {
   swap = swap || {}
   const {
     orderId = '', sendSymbol = '', depositAddress = '', receiveSymbol = '', receiveAddress = '',
-    sendAmount, receiveAmount, rate, orderStatus = '', refundAddress = '', isFixedPrice,
+    sendAmount, receiveAmount, rate, orderStatus = '', refundAddress = '', isFixedPrice, 
+    sendAsset: { ERC20 }
```

Comment:
You already have `bip21Prefix` in `sendAsset` so you could omit the extra selector